src/Views/PartitioningView.vala

```diff
@@ -225,7 +225,6 @@ public class Installer.PartitioningView : AbstractInstallerView {
             case Distinst.PartitionTable.MSDOS:
                 break;
             case Distinst.PartitionTable.GPT:
-                required |= Defined.EFI;
```

Comment:
Is there a way to detect if the system is in EFI mode to enforce this requirement when it makes sense?

Reply:
This PR is to exclude the requirement (make it optional) for an EFI system partition (ESP) for custom install mode in any case.

I presume anyone choosing to do a custom install would be familiar with the requirement for an ESP and will add one with the partitioning tool if they are, say, installing only elementary OS on their machine in custom mode.

Reply:
Yeah good point. For some reason when I wrote this in my head this was also for regular installs as well